portal/client/src/components/DetailsView/DetailsView.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+const DetailsView = ({studyData={}}) => {
+
+    return (
+        <div className="col-8">
```

Comment:
make these to reusable sub components

Reply:
lets keep the stuctural classes out of this components